rpc/js/src/dial.ts

```diff
@@ -474,6 +474,23 @@ export async function dialWebRTC(
       const offerDesc = await pc.createOffer({});
 
       let iceComplete = false;
+      let numCallUpdates = 0;
+      let maxCallUpdateDuration = 0;
+      let totalCallUpdateDuration = 0;
+
+      pc.addEventListener('iceconnectionstatechange', () => {
+        if (pc.iceConnectionState !== 'completed' || numCallUpdates === 0) {
```

Comment:
Typescript has a similar situation to Go here: we never seem to reach `completed`, only `connected`. I've also found that in typescript, we can reach a state of `connected` without ever fully submitting a local ICE candidate to the DB (`numCallUpdates === 0`). I'm not sure it's possible to write code similar to [what we have in Go](https://github.com/viamrobotics/goutils/blob/main/rpc/wrtc_client.go#L163-L176), as I believe this TS code does not actually run in another thread and can't "wait" for more caller updates to finish.